TACC/Core-CMS-Custom
# fix(demdata-cms): rename, update, fix

## Overview

Make DemData CMS be maintained and built in this repo, instead of https://github.com/TACC/Core-CMS-Resources/.

## Related

- based off of #161
- might require https://github.com/TACC/Core-CMS/pull/606

## Changes

- **removed** cruft files
- **changed** cms docker image tag
- **added** settings and default secrets
- **changed** dir names to `demdata_cms` (for Django 3)
- **changed** settings
- **added** project images

## Testing

1. Deploy using cms image tag of a build of this branch.
2. Open https://pprd.democratizing-site.tacc.utexas.edu/.
3. Verify it looks like Core-CMS v3.10-ish, but with:
    - DemData logo
    - **no** logos in branding bar (top part of header)
4. Re-deploy using cms image tag that production uses.
5. Verify it looks like Core-CMS v3.10-ish, but with:
    - DemData logo
    - **two** logos in branding bar (top part of header)

## UI

### _Old_ Build, _Old_ Styles, _Test_ Content

**Skipped**, based on more useful tests succeeding.

| local | staging | prod |
| - | - | - |
| — | — | ⊗ |

### _New_ Build, _Old_ Styles, _Test_ Content

| local | staging | prod |
| - | - | - |
| ![pprd, new build, old styles](https://github.com/TACC/Core-CMS-Custom/assets/62723358/ccebaae9-5e90-49fd-8ab9-42412eaeaced) | ![local, new build, old styles](https://github.com/TACC/Core-CMS-Custom/assets/62723358/bcb04168-6bb0-457b-8d7e-7159412699c6) | ⊗ |

### _New_ Build, _New_ Styles, _Test_ Content

**Tested** successful, but I lost the screenshots.

| local | staging | prod |
| - | - | - |
| ✓ | ✓ | ⊗ |

### _Prod_ Content

| staging & prod |
| - |
| ![demdata](https://github.com/TACC/Core-CMS-Custom/assets/62723358/d592a588-e493-4fea-a9ac-1ed63d2fae86) |
